### DISCUSSION TITLE
Abyss Diving Suit does not consume oxygen tanks slower

### DISCUSSION THREAD

**Discussion by Smrtonosan** [FakeFishGames/Barotrauma#15150] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Abyss Diving Suit description states that it uses up less oxygen, and the wiki states that oxygen tanks last longer inside an abyss suit.
![20241114144452_1](https://github.com/user-attachments/assets/5cd16ea3-1673-4f15-b795-c19deb96d5f4)
However in-game it doesn't appear to work.

### Reproduction steps

1. Load Submarine in sub maker.
2. Spawn two assistants.
3. Spawn an abyss diving suit, regular diving suit and two oxygen tanks. (put oxygen into the suits)
4. Equip the abyss diving suit on one bot and then the regular diving suit on the other.
5. See that the abyss diving suit will consume oxygen at the same rate and that it will always have less oxygen than the regular diving suit because it was equipped first, and as such it consumes oxygen at the same rate as the regular diving suit.

Thinking that there is a mechanic where maybe suits consumed more oxygen at bigger depths I dove to the depth of 5700 meters but the same test resulted the same results even outside the submarine? Could be an editor thing

I discovered this in multiplayer where the abyss diving suit was consuming the inserted oxygen and oxygenite tanks suspiciously quickly, as such I will mark it as happening in multiplayer as well.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by JeroonBosch** [comment]
Thank you for your report. Abyss Suits need to have their description adjusted as they indeed don't consume oxygen tanks any slower. Having oxygen depletion rates vary with both the suit and oxygen tanks (with quality further affecting it) is confusing, so we're opting for only the oxygen/oxygenite tank affecting oxygen duration.
